gems/sorbet-runtime/test/types/edge_cases.rb

```diff
@@ -21,19 +21,37 @@ def ==(other)
       extend T::Helpers
       sig {params(foo: String).returns(String)}
       attr_writer :foo
-      sig {params(bar: Integer).returns(Integer)}
+      sig { returns(Integer)}
```

Comment:
This `sig` had always been wrong, since the documentation clearly states to use the reader form for accessor sigs.